canonical-web-and-design/ubuntu.com
# 9555: Add video to financial-services

## Done

Update the page according to the copydoc mentioned on #9555

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/financial-services
- Check the changes match the [copydoc](https://docs.google.com/document/d/1SEd2nV0GplBTG4lZ9RuhjNXrphU22nGP4Fpgd5g_9BQ/edit#)


## Issue / Card

Fixes #9555
